### ISSUE TITLE
Creating a python module for anki_testing

### ISSUE THREAD

**Issue by luc-vocab** [krassowski/anki_testing#4]
The current way that addons use anki_testing is a bit bespoke and could be simplified. Here's what i'm suggesting:
- we create a python module which contains anki_testing code and pulls in necessary dependencies (both anki python modules, pyqt5 and anything else which is required)
- addon creators would simply pip install anki_testing in their travis CI config, and then simply run pytest.
- there would be no need to check out the anki_testing repository
- builds would be faster and less confusing, and more aligned with the way travis CI works.

do you have any thoughts on this ?

**Comment by krassowski** [comment]
Yes, this is a great idea, just a little more demanding on maintenance as requires someone to do the actual releases - or someone to write up a bot for that.
